test/queryselector.spec.ts

```diff
@@ -312,4 +359,100 @@ describe('querySelector', function () {
       expect(second).toEqual([]);
     });
   });
+
+  // this is the same tests for `$$eval` and `$$` as above, but with a handler
```

Comment:
nit: comments should be sentences

```suggestion
  // This is the same tests for `$$eval` and `$$` as above, but with a handler
```

Reply:
Done.